tests/e2e_regression/run.sh

```diff
@@ -69,18 +69,19 @@ testCases=(
 nCases=${#testCases[@]}
 
 # Kill background processes on exit. (In our case the indexer API server.)
+# todo: this doesn't work on MacOS. 
```

Comment:
Do you know what the culprit is? Longer term, it might be worth coding this up in Go or python I guess ...

Reply:
Er not off the top of my head; but I'll try running it with `bash` from now on instead of `zsh` which should hopefully resolve it.